src/test/java/org/folio/search/support/base/BaseConsortiumIntegrationTest.java

```diff
@@ -33,38 +23,6 @@ protected static void removeTenant() {
     removeTenant(CENTRAL_TENANT_ID);
   }
 
-  @SneakyThrows
-  protected static void setUpTenant(String tenantName, Instance... instances) {
-    setUpTenant(tenantName, emptyList(), instances);
-  }
-
-  @SneakyThrows
-  protected static void setUpTenant(String tenantName, List<ResultMatcher> matchers, Instance... instances) {
-    setUpTenant(tenantName, instanceSearchPath(), () -> { }, asList(instances), instances.length, matchers,
-      instance -> inventoryApi.createInstance(tenantName, instance));
-  }
-
-  @SneakyThrows
-  protected static <T> void setUpTenant(String tenant, String validationPath, Runnable postInitAction,
-                                        List<T> records, Integer expectedCount, List<ResultMatcher> matchers,
-                                        Consumer<T> consumer) {
-    enableTenant(tenant);
-    postInitAction.run();
-    saveRecords(tenant, validationPath, records, expectedCount, matchers, consumer);
-  }
-
-  @SneakyThrows
-  protected static void enableTenant(String tenant) {
-    var tenantAttributes = new TenantAttributes().moduleTo("mod-search");
-    tenantAttributes.addParametersItem(new Parameter("centralTenantId").value(CENTRAL_TENANT_ID));
-
-    mockMvc.perform(post("/_/tenant", randomId())
-        .content(asJsonString(tenantAttributes))
-        .headers(defaultHeaders(tenant))
-        .contentType(APPLICATION_JSON))
-      .andExpect(status().isNoContent());
-  }
```

Comment:
What is the reason for these changed?

Reply:
Code deduplication: BaseIntegrationTest already contains the same methods or methods which closely similair.